GETTING_STARTED.md

```diff
@@ -0,0 +1,416 @@
+# Getting started
+
+In this document you will find a step-by-step guide on how to get Eunomia running in a local Minikube cluster.
+
+The resources Eunomia, the GitOps controller, is watching and how it is done will be discussed in general.
+
+You will run two examples:
+- Eunomia acting upon local GitOpsConfig changes,
+- Eunomia watching for changes in a remote repository and acting upon them.
+
+## Prerequisites
+
+In order to build Eunomia, you'll need the following:
+
+- [Git](https://git-scm.com/downloads)
+- [Go 1.12+](https://golang.org/dl/)
+- [Docker](https://docs.docker.com/install/)
+- [Operator SDK v0.8.1](https://github.com/operator-framework/operator-sdk/blob/v0.8.1/doc/user/install-operator-sdk.md)
+- Access to a Kubernetes cluster
+  - [Minikube](https://kubernetes.io/docs/setup/minikube/) (for local development and testing)
+    - [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+  - [Minishift](https://www.okd.io/minishift/) (for local development and testing, optional)
+    - [oc](https://docs.okd.io/latest/cli_reference/get_started_cli.html#installing-the-cli)
+
+### Installing on a Mac
+
+All the components (except operator-sdk v0.8.1) can be easily installed via [Homebrew](https://brew.sh/) on a Mac:
+
+```shell
+brew install git
+brew install go@1.12
+brew install docker
+brew install minikube
+brew install kubectl
+brew install minishift # (optional)
+brew install openshift-cli # (optional)
+```
+
+Unfortunately, operator-sdk in version 0.8.1 isn't available via brew so you have to install it from GitHub release as described [on this page](https://github.com/operator-framework/operator-sdk/blob/v0.8.1/doc/user/install-operator-sdk.md).
+
+### Checkout your fork
+
+Your own remote fork will be needed later to present how Eunomia adresses changes pushed to a repo being watched.
+
+To check out this repository:
+
+1. Create a [fork](https://help.github.com/en/articles/fork-a-repo) of this repo
+2. Create the directories and clone your fork
+
+```
+mkdir -p $GOPATH/src/github.com/KohlsTechnology
+cd $GOPATH/src/github.com/KohlsTechnology
+git clone https://github.com/<YOUR FORK>/eunomia.git
+cd eunomia
+```
+
+## Deploy Eunomia to a Minikube cluster
+
+### Start a local Minikube cluster
+
+Minikube is a minimal kubernetes cluster run in a virtual machine in Virtualbox.
```

Comment:
Minikube needs to be configured for virtualbox. We should have those instructions.
This is also great for a Mac, what about other platforms?

Reply:
Minikube by default automatically selects the virtualbox driver (I missed virtualbox in the prerequisites :/ ). But I agree - better add an explicit instruction.

The instructions for Mac already were in DEVELOPMENT.md, I just updated them to be sure that everything works and moved it to GETTING_STARTED.md. I'll add a general setup for Linux linking to vendor instructions - I could add code snippets for yum, apt, dnf, or pacman but it would be too much testing